sechub-administration/src/test/java/com/mercedesbenz/sechub/domain/administration/user/UserRestControllerTest.java

```diff
@@ -1,11 +1,12 @@
 // SPDX-License-Identifier: MIT
 package com.mercedesbenz.sechub.domain.administration.user;
 
+import static com.mercedesbenz.sechub.test.RestDocPathParameter.EMAIL_ADDRESS;
 import static com.mercedesbenz.sechub.test.SecHubTestURLBuilder.https;
 import static org.hamcrest.CoreMatchers.equalTo;
-import static org.mockito.Mockito.verify;
-import static org.mockito.Mockito.when;
-import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.mockito.Mockito.*;
+import static org.springframework.security.test.web.servlet.request.SecurityMockMvcRequestPostProcessors.csrf;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
```

Comment:
Not your fault, but please rename the class to
`UserRestControllerSpringBootTest`